cfn-resources/trigger/test/inputs_1_create.template.json

```diff
@@ -12,7 +12,8 @@
     "FullDocument": "true",
     "FullDocumentBeforeChange": "false",
     "Unordered": "false",
-    "SkipCatchupEvents": "false"
+    "SkipCatchupEvents": "false",
+    "Match" : "{\"$and\":[{\"fullDocument.availability.value\":\"AVAILABLE\"},{\"$ne\":[\"fullDocument.retailPrice\",\"fullDocumentBeforeChange.retailPrice\"]}]}"
```

Comment:
Can you also update the file in the example folder?

Reply:
+1 on an example to help our users here, both for CFN & CDK.

Deferring my LGTM to @andreaangiolillo here for those tasks